spec/acceptance/management_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper_acceptance'
+
+describe 'rancher::management' do
+  context 'puppet apply' do
+    it 'is expected to apply and be idempotent' do
+      pp = <<-EOS
+class { 'rancher::management':
+  rancher_manager_port => '8080',
+}
+EOS
+
+      apply_manifest(pp, :catch_failures => true)
+      apply_manifest(pp, :catch_changes => true)
+    end
+
+    describe docker_container('rancher/server') do
+      it { is_expected.to be_running }
+    end
+
+    context 'is expected to be running on the default port' do
+      before :each do
+        sleep 5 # Give it time to start.
+      end
+
+      describe command('curl 0.0.0.0:8080') do
+        its(:stdout) { is_expected.to match /self.*0.0.0.0:8080/ }
```

Comment:
Isn't this better to change this to a test to check the front page is actually working?

the previous `its(:stdout) { should match /\<title>rancher<\/title>/ }` would actually check that the app is running and the HTML is there, this might have an issue if the response is returned but 500 or such.